shared/versions.yml

```diff
@@ -10,110 +10,110 @@ cert_manager:
     repository: quay.io/jetstack/cert-manager-controller
     tag: v0.6.1
   generated:
-    image: gcr.io/gpii-common-prd/quay.io/jetstack/cert-manager-controller
+    repository: gcr.io/gpii2test-common-stg/quay.io/jetstack/cert-manager-controller
```

Comment:
This value is incorrect (except for my testing), but version-updater will fix it once everything is merged.